### DISCUSSION TITLE
深绿色与浅绿色哪个表示更加努力比较政治正确？

### DISCUSSION THREAD

**Discussion by fanhan-inside** [fanhan-inside/fanhan-inside.github.io#49] [General]
简单说，GitHub有「夜间模式」「暗黑主题」，其中反应提交次数的颜色深浅刚好相反，似乎光天化日朗朗乾坤之下肤色越深越革命，而深更半夜黑灯瞎火的时候肤色越浅越亮眼。

![20201219-001_001](https://user-images.githubusercontent.com/41445691/135496506-ee1b522d-f09e-4c1d-a475-f0aaa2d7c7e3.png)

![20201219-001_002a](https://user-images.githubusercontent.com/41445691/135496520-c28e8d63-c68c-4fac-b2ef-08bb9362ad2a.png)
![20201219-001_003a](https://user-images.githubusercontent.com/41445691/135496524-9d7f108e-abc0-4c53-80f2-69df1338280b.png)

![20201219-001_002b](https://user-images.githubusercontent.com/41445691/135496521-1bcab67e-098c-403f-952d-750d82383e2d.png)
![20201219-001_003b](https://user-images.githubusercontent.com/41445691/135496528-16b27632-839f-4be0-af5b-04feeb86dd57.png)

那么，在「世界上最诚实的人写下六行字，就一定能从中找出足够的罪证，将其打翻在地再踏上一只脚令其永世不得翻身」的政治规矩和组织纪律之下，各位打工人都是怎么向根红苗正忠君爱国的富二代领导汇报的呢？

参考美帝灯塔国政治笑话「被指责为种族歧视的教练公开对球员讲话：“现在我眼里既没有黑人也没有白人，只有绿人。好了，深绿色的球员站那边，浅绿色的球员站这边”」，这个严肃的问题，比「左脚还是右脚先进门」更敏感。

**Comment by CESTBON5146** [comment]
苏丹加哈里发两位一体，让我想起了《赤色黎明》中陈克被评价的情节，‘赤色’中说中国自古以来最为推崇的领导人模版就是这种老师+领袖的双重身份

**Reply by fanhan-inside** [reply]
> 苏丹加哈里发两位一体，让我想起了《赤色黎明》中陈克被评价的情节，‘赤色’中说中国自古以来最为推崇的领导人模版就是这种老师+领袖的双重身份

确切的说是柏拉图以来最为推崇的领导人模板就是「哲人王」，我中华兲朝上国自古以来什么样，不能通过西学东渐以来与时俱进的当代汉语普通话语境里运用繁荣的简体中文互联网上传媒大亨推送到眼皮底下的公开信息编纂的文艺作品判断。
